scipy/interpolate/polyint.py

```diff
@@ -571,7 +570,7 @@ def add_xi(self, xi, yi=None):
         old_wi = self.wi
         self.wi = np.zeros(self.n)
         self.wi[:old_n] = old_wi
-        for j in xrange(old_n,self.n):
+        for j in range(old_n,self.n):
```

Comment:
nit, space after the comma

Reply:
In a separate pull request, just pass the whole codebase thru psf/black.

Reply:
In an ideal world it would be nice to have that. However, this would cause havoc to the codebase as there would be so many files touched, creating issues for in progress PR's, churn, etc.  Making all those changes to the entire codebase would cause more harm than good. It's better to make such changes as coincidental alterations (such as this) are made.